classification/extractor/t/test_extractor.py

```diff
@@ -2,12 +2,13 @@
 import sys
 
 from classification.extractor.low_level_data_extractor import make_low_level_data_file
+from filehandler.handle_path import get_absolute_path
 
 def test_song_data_extraction():
+    filename = get_absolute_path("classification/extractor/t/test_extractor/8376-1-1_Demolition_Man_proud_music_preview"
+                                 ".wav")
```

Comment:
I think this line is too long. Why is it two strings?

Reply:
:+1: to Casper's comment.